src/cmdline_config.rs

```diff
@@ -39,13 +39,13 @@ pub fn config_from_cmdline() -> Config {
                            .arg(Arg::with_name("port")
                                 .short("p")
                                 .long("--port")
-                                .help("Port to run TCP service on")
+                                .help("Port to run TCP service on (default: 10110)")
                                 .takes_value(true)
                                 .value_name("PORT"))
                            .arg(Arg::with_name("interface")
                                 .short("n")
                                 .long("--network-interface")
-                                .help("Bind specific network interface")
+                                .help("Bind specific network interface (default: all)")
```

Comment:
this change is isn't part of the logical change of this patch. I know it's just one line but I'm strict on "one logical change == one patch" rule. :) Feel free to put both these help improvements into the same patch on top of these two.